applications/ParticleMechanicsApplication/trilinos_extension/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+include_directories(
+  ${CMAKE_SOURCE_DIR}/kratos
+  ${CMAKE_SOURCE_DIR}/applications/ParticleMechanicsApplication
+  ${CMAKE_SOURCE_DIR}/applications/TrilinosApplication
+)
+
+if (NOT TRILINOS_FOUND)
+  find_package(TRILINOS REQUIRED)
```

Comment:
This should be done differently, check the cmakelists of the MPI extension of the MappingApp